apps/sms/js/thread_ui.js

```diff
@@ -2531,10 +2531,10 @@ var ThreadUI = {
       // eg. type "a", then delete it.
       // Always remove the the existing results.
       this.toggleRecipientSuggestions();
-      return;
+      return Promise.resolve(null);
```

Comment:
I removed toggleRecipientSuggestions so that it's run in listContacts now (I moved it inside listContacts as well).

I think this is not an exception because this happens in normal use of the app; do you think we should just not call searchContact in case there is no filterValue, and remove the suggestion panel earlier in the code?

i like the current code (as with my patch), tell me what you think.